shell/platform/common/accessibility_bridge.h

```diff
@@ -176,7 +204,7 @@ class AccessibilityBridge
   std::unordered_map<AccessibilityNodeId,
                      std::shared_ptr<FlutterPlatformNodeDelegate>>
       id_wrapper_map_;
-  ui::AXTree tree_;
+  std::unique_ptr<ui::AXTree> tree_;
```

Comment:
Changed this to a pointer so we can return a pointer in the implementation for AX Tree Manager.